.github/label-commenter-config.yml

```diff
@@ -0,0 +1,52 @@
+comment:
+  # header: "Please note the following requirement:"
+  footer: "\
+    ---\n\n
+    > &nbsp; &#9; &nbsp; &#9;  &nbsp; &#9;  &nbsp; &#9; Be sure to [join the community](http://slack.layer5.io), if you haven't yet and please leave a :star: [star on the project](../stargazers) :smile:
+    "
+
+labels:
+  - name: issue/design required
+    labeled:
+      issue:
+        body: This issue has been labeled with 'design-required'. Note that prior to commencing on implementation, a design specification needs to be created and reviewed for approval. See [Creating a Functional Specification](https://docs.google.com/document/d/1RP3IWLc-MiQS-QYasqCoVuCH7--G87p5ezE5f_nOzB8/edit?usp=sharing) to create a design spec.
+        action: open
+  - name: issue/remind
+    labeled:
+      issue:
+        body: Checking in... it has been awhile since we&#39;ve heard from you on this issue. Are you still working on it? Please let us know and please don&#39;t hesitate to contact a [MeshMate](https://layer5.io/community/meshmates/) or any other [community member](https://layer5.io/community/members) for assistance.
+        action: open
+      pr: 
+        body: Checking in... it has been awhile since we&#39;ve heard from you on this issue. Are you still working on it? Please let us know and please don&#39;t hesitate to contact a [MeshMate](https://layer5.io/community/meshmates/) or any other [community member](https://layer5.io/community/members) for assistance.
+        action: open
+  - name: issue/dco
+    labeled:
+      issue:
```

Comment:
@anubhav201241: We don't need to trigger `issue/dco` workflow for the issue, pl remove [these lines](https://github.com/layer5io/discuss-board/blob/c6315659080f6b0db884b91ad9cbd7dccf511fd2/.github/label-commenter-config.yml#L24-L28).

Reply:
Hey @amino19 any reason why are we moving this issue body because we do have this same kind of structure in Meshery https://github.com/meshery/meshery/blob/master/.github/label-commenter-config.yml#L23-L28. And also I have verified that if this issue body is missing then the github action bot do not trigger in the PR.

Reply:
Hey @anubhav201241 I think we can keep these lines.

Reply:
@sayantan1413: I think we need this for pull requests only. Is there any issue need to be label as `issue/dco`?

Reply:
I was thinking to keep it consistent with Meshery but what you are suggesting also makes sense!